pkg/microservice/aslan/core/service/handler/router.go

```diff
@@ -42,6 +42,7 @@ func (*Router) Inject(router *gin.RouterGroup) {
 		helm.POST("/services", CreateOrUpdateHelmService)
 		helm.POST("/services/bulk", CreateOrUpdateBulkHelmServices)
 		helm.PUT("/:productName", UpdateHelmService)
+		helm.PUT("/services/releaseNaming", HelmReleaseNaming)
```

Comment:
Does the new API require permission control?

Reply:
https://github.com/koderover/zadig/pull/1291/files

Reply:
currently we don't have permission control on apis about service templates

Reply:
permission control added